tests/view_tests/tests/test_debug.py

```diff
@@ -57,7 +57,7 @@ def __call__(self):
 
 
 @override_settings(DEBUG=True, ROOT_URLCONF='view_tests.urls')
-class DebugViewTests(LoggingCaptureMixin, SimpleTestCase):
+class DebugViewTests(SimpleTestCase):
```

Comment:
oh right, I missed that too I thought I had added it to the debug class and you had removed it from there. late night coding.. =(